api/tax_calculator/tax_brackets.py

```diff
@@ -1,26 +1,24 @@
 import os
-import json
-from flask import jsonify
 
+from api import utils
 
-brackets_dir = os.path.join(os.path.dirname(__file__), 'fixtures')
+SUPPORTED_TAX_BRACKET_YEARS = [2019, 2020]
 
+BRACKETS_DIRECTORY = os.path.join(os.path.dirname(__file__), 'fixtures')
 
-def _open_config(filename):
 
-    with open(filename) as config_file:
-        json_contents = json.load(config_file)
-        config_file.close()
+def get_all_tax_brackets():
+    tax_brackets = {}
+    for year in SUPPORTED_TAX_BRACKET_YEARS:
+        tax_brackets[str(year)] = get_tax_brackets(year)
 
-    return json_contents
 
-
-def get_tax_brackets(tax_year='2020'):
-    filename = f'tax-brackets--{tax_year}.json'
-    file_with_path = os.path.join(brackets_dir, filename)
+def get_tax_brackets(tax_year=2020):
+    filename = f'tax-brackets-{tax_year}.json'
+    file_with_path = os.path.join(BRACKETS_DIRECTORY, filename)
     try:
-        return _open_config(file_with_path)
+        return utils.get_json_contents_from_resource_file(file_with_path)
```

Comment:
I also don't know if this is worth abstracting out. You save two lines of easily readable code at the cost of having to go to a separate part of the codebase to find out what's going on.

Reply:
But both assignments use the same functionality? Hmm then does that mean we should take everything out of utils and put them into each directory's model (`autonomous_car.py` & `tax_calculator.py`)